core/tx_pool.go

```diff
@@ -173,6 +171,10 @@ func (config *TxPoolConfig) sanitize() TxPoolConfig {
 		log.Warn("Sanitizing invalid txpool price limit", "provided", conf.PriceLimit, "updated", DefaultTxPoolConfig.PriceLimit)
 		conf.PriceLimit = DefaultTxPoolConfig.PriceLimit
 	}
+	if isQuorum && (conf.PriceLimit == DefaultTxPoolConfig.PriceLimit) {
+		log.Warn("Sanitizing invalid txpool price limit for Quorum", "provided", conf.PriceLimit, "updated 0")
```

Comment:
There's a small issue with the log statement. An even number of parameters are expected after the log message - the string "updated 0" is expected to be followed by a variable which is to be printed.
Because the variable is missing, the log statement is catching that and printing:
    LOG15_ERROR="Normalized odd number of arguments by adding nil"

Reply:
Ok. WIll make the change.